### PR TITLE
increase wait period

### PR DESCRIPTION
**Pull request by LudwigKnuepfer**
otherwise the loop won't run

### CODE REVIEW

File: test_irq/main.c [hunk 1]
```diff
@@ -41,7 +41,7 @@ int main(void)
     printf("busy_thread created\n");
 
     printf("hwtimer_wait()\n");
-    hwtimer_wait(HWTIMER_TICKS(10));
+    hwtimer_wait(HWTIMER_TICKS(100000));
```
Comment (mehlis) [inline test_irq/main.c:44]: so you want to wait for 100ms?

Comment (LudwigKnuepfer) [inline test_irq/main.c:44]: Did you test the test case?

Comment (mehlis) [inline test_irq/main.c:44]: I can imagine, that waiting 10ms is to short for "something" to happen.
before:

```
busy_thread created
hwtimer_wait()
main: return
busy_thread starting
i: 0
j: 0
k: 23
success
```

after:

```
busy_thread created
hwtimer_wait()
busy_thread starting
main: return
i: 37123626
j: 37123626
k: 37123626
success
```

**Comment by mehlis** [comment]
@LudwigOrtmann please reopen this PR for `RIOT/tests`

**Comment by OlegHahm** [comment]
It's already in there.

**Pull request closed without merging**